cvat/schema.yml

```diff
@@ -7867,11 +7867,17 @@ components:
           default: 0
         completed:
           type: integer
-          default: 0
+          nullable: true
+        validation:
+          type: integer
+          nullable: true
```

Comment:
Do these fields have to be nullable? Doesn't seem like a null value would be very useful.

Reply:
Answered above